SimpleWAWS/Code/BackgroundQueueManager.cs

```diff
@@ -61,26 +63,31 @@ public void DeleteResourceGroup(ResourceGroup resourceGroup)
 
         private async Task<ResourceGroup> LogActiveUsageStatistics(ResourceGroup resourceGroup)
         {
-            try
+            if (resourceGroup.SubscriptionType == Subscription.SubscriptionType.AppService)
             {
-                var site = resourceGroup.Sites.First(s => s.IsSimpleWAWSOriginalSite);
-                var credentials = new NetworkCredential(site.PublishingUserName, site.PublishingPassword);
-                var zipManager = new RemoteZipManager(site.ScmUrl + "zip/", credentials);
-                using (var httpContentStream = await zipManager.GetZipFileStreamAsync("LogFiles/http/RawLogs"))
+                try
                 {
-                    await StorageHelper.UploadBlob(resourceGroup.ResourceUniqueId, httpContentStream);
+                    foreach (var site in resourceGroup.Sites)
```

Comment:
There will still be only one file but the resource name now includes the sitename. 
We'll have to parse the resourcename to know what site it is for before uploading to DB.